### PR TITLE
Syndicate maid outfit + the unatomic Teshari eyepatch fix

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request

Tin, adds a Syndicate maid outfit, find it in the premium section of the Syndivend. I also used this as an excuse to fix Teshari eyepatches.

## How This Contributes To The Skyrat Roleplay Experience

It probably doesn't but atleast 3 people asked me for it. Also eyepatches being fixed for birds is nice, y'know.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
maid spin

</details>

## Changelog

:cl:
imageadd: The Syndicate has gotten a shipment of Maid outfits.
fix: Teshari's eyepatches should work and be able to swap eyes now.
/:cl:

### CODE REVIEW

**Comment by Stalkeros2** [comment]
Oh Jod.

**Comment by Stalkeros2** [comment]
It should really be a buyable bundle in the uplink, along with a lot of soap/cleaning supplies, soap grenades, and that soap spawner syndie bomb. ~~And a Kitchen Gun™️.~~

**Comment by Paxilmaniac** [comment]
mfer down bad enough that he posted two comments in a row

File: modular_skyrat/modules/mapping/code/wardrobes.dm [hunk 1]
```diff
@@ -27,7 +27,10 @@
 					/obj/item/clothing/shoes/combat/coldres = 2)
 	premium = list(/obj/item/knife/combat/survival = 1,
 					/obj/item/storage/fancy/cigarettes/cigpack_syndicate = 5,
-					/obj/item/clothing/gloves/combat = 3)
+					/obj/item/clothing/gloves/combat = 3,
+					/obj/item/clothing/under/syndicate/skyrat/maid = 5,
+					/obj/item/clothing/gloves/combat/maid = 5,
+					/obj/item/clothing/head/maidheadband/syndicate = 5)
```
Comment (Jolly-66) [inline modular_skyrat/modules/mapping/code/wardrobes.dm]: Become codeowner for thos module
Check notifs
Question why I'm tagged
mfw

Comment (Ebin-Halcyon) [reply modular_skyrat/modules/mapping/code/wardrobes.dm]: Mogged.

**Comment by Ebin-Halcyon** [comment]
> It should really be a buyable bundle in the uplink, along with a lot of soap/cleaning supplies, soap grenades, and that soap spawner syndie bomb. ~And a Kitchen Gun™️.~

idk if you wanna do that, be my guest.

File: modular_skyrat/modules/mapping/code/wardrobes.dm (hunk 1 above)
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/code/wardrobes.dm]: Try to reduce the indentation of this whole list too, just, in general, formatting it like all the other lists with a ton of entries usually are. Hit me up on Discord if you're not sure.
```suggestion
					/obj/item/clothing/gloves/combat = 3,
					/obj/item/clothing/under/syndicate/skyrat/maid = 5,
					/obj/item/clothing/gloves/combat/maid = 5,
					/obj/item/clothing/head/maidheadband/syndicate = 5,
	)
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Other than that, thank you for fixing the eyepatch bug for Tesharis, I didn't know they had a file for that, otherwise I would've done it myself when fixing that one refactor.

File: modular_skyrat/modules/mapping/code/wardrobes.dm [hunk 2]
```diff
@@ -9,25 +9,29 @@
 	product_ads = "Put a Donk on it!;Aim, Style, Shoot!;Brigged for wearing the best!"
 	vend_reply = "Thank you for using the SynDrobe!"
 	products = list(/obj/item/clothing/under/syndicate/skyrat/tactical = 3,
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/code/wardrobes.dm]:
```suggestion
	products = list(
		/obj/item/clothing/under/syndicate/skyrat/tactical = 3,
```

File: modular_skyrat/modules/mapping/code/wardrobes.dm [hunk 3]
```diff
@@ -9,25 +9,29 @@
...
+		/obj/item/clothing/under/syndicate/bloodred/sleepytime = 3,
+		/obj/item/clothing/under/syndicate/sniper = 3,
+		/obj/item/clothing/under/syndicate/camo = 3,
+		/obj/item/clothing/under/syndicate/combat = 3,
+		/obj/item/clothing/shoes/combat = 3,
+		/obj/item/clothing/mask/gas/syndicate = 3,
+		/obj/item/clothing/mask/gas/sechailer/syndicate = 3,
+		/obj/item/clothing/suit/hooded/wintercoat/syndicate = 5,
+		/obj/item/clothing/head/soft/sec/syndicate = 3,
+		/obj/item/clothing/head/beret/sec/syndicate = 3)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/code/wardrobes.dm]:
```suggestion
		/obj/item/clothing/head/beret/sec/syndicate = 3,
	)
```

File: modular_skyrat/modules/mapping/code/wardrobes.dm [hunk 4]
```diff
@@ -9,25 +9,29 @@
...
 	contraband = list(/obj/item/knife/combat = 1,
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/code/wardrobes.dm]:
```suggestion
	contraband = list(
		/obj/item/knife/combat = 1,
```

File: modular_skyrat/modules/mapping/code/wardrobes.dm [hunk 5]
```diff
@@ -9,25 +9,29 @@
...
-					/obj/item/clothing/under/syndicate/coldres = 2,
-					/obj/item/clothing/shoes/combat/coldres = 2)
+		/obj/item/clothing/under/syndicate/coldres = 2,
+		/obj/item/clothing/shoes/combat/coldres = 2)
+
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/code/wardrobes.dm]:
```suggestion
		/obj/item/clothing/shoes/combat/coldres = 2,
	)
```

File: modular_skyrat/modules/mapping/code/wardrobes.dm [hunk 6]
```diff
@@ -9,25 +9,29 @@
...
 	premium = list(/obj/item/knife/combat/survival = 1,
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/code/wardrobes.dm]:
```suggestion
	premium = list(
		/obj/item/knife/combat/survival = 1,
```

File: modular_skyrat/modules/mapping/code/wardrobes.dm [hunk 7]
```diff
@@ -9,25 +9,29 @@
...
-					/obj/item/storage/fancy/cigarettes/cigpack_syndicate = 5,
-					/obj/item/clothing/gloves/combat = 3)
+		/obj/item/storage/fancy/cigarettes/cigpack_syndicate = 5,
+		/obj/item/clothing/gloves/combat = 3,
+		/obj/item/clothing/under/syndicate/skyrat/maid = 5,
+		/obj/item/clothing/gloves/combat/maid = 5,
+		/obj/item/clothing/head/maidheadband/syndicate = 5)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/code/wardrobes.dm]:
```suggestion
		/obj/item/clothing/head/maidheadband/syndicate = 5,
	)
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Should be the last things that need changed.

**Review by GoldenAlpharex — APPROVED** [review]
Should hopefully be good to go now!

**Pull request merged**